docs/develop/go/temporal-nexus.mdx

```diff
@@ -542,12 +544,9 @@ tcld nexus endpoint create \
 The `--allow-namespace` is used to build an Endpoint allowlist of caller Namespaces that can use the Nexus Endpoint, as described in Runtime Access Control.
 
 Alternatively, you can create a Nexus Endpoint through the UI: [https://cloud.temporal.io/nexus](https://cloud.temporal.io/nexus).
-You can also create a Nexus endpoint through the UI in the development server.
 
 ### Run Workers Connected to Temporal Cloud with TLS certificates
 
-[View the source code](https://github.com/temporalio/samples-go/tree/main/nexus) in the context of the rest of the application code.
-
```

Comment:
This seemed out of place; it's not linking to code specific to this cloud section.